src/components/Radio/index.tsx

```diff
@@ -3,9 +3,24 @@ import styled from '@emotion/styled'
 import type { StyledProps } from '@types'
 
 export interface RadioProps extends HTMLAttributes<HTMLFormElement> {
+  /** Radio 컴포넌트의 이름을 정합니다.(input name에 사용)
+   * @type string
+   */
   formName: string
+
+  /** Radio 컴포넌트의 들어갈 요소들을 정합니다
+   * @type { code: string, name: string } []
+   */
   items: { code: string; name: string }[]
```

Comment:
```suggestion
  /** Radio 컴포넌트에 보여질 옵션들을 정합니다
   * @type { code: string, name: string } []
   */
  items: { code: string; name: string }[]
```
요기도 좀 더 명확하게 하면 좋을 것 같아요!